templates/layout.html

```diff
@@ -6,8 +6,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>{% block title %}{% endblock %}</title>
-    <link rel="stylesheet" type="text/css" href="../static/css/bootstrap.min.css">

```

Comment:
leave the spectrum css file as it is (or reincorporate the old spectrum.css file in next commit) coz i modified it a bit. so in case of downloading the latest file might not have the same modifications etc.